service/history/decisionHandler.go

```diff
@@ -295,17 +285,17 @@ func (handler *decisionHandlerImpl) handleDecisionTaskCompleted(
 	clientFeatureVersion := call.Header(common.FeatureVersionHeaderName)
 	clientImpl := call.Header(common.ClientImplHeaderName)
 
-	context, release, err0 := handler.historyCache.getOrCreateWorkflowExecutionWithTimeout(ctx, domainID, workflowExecution)
-	if err0 != nil {
-		return nil, err0
+	context, release, err := handler.historyCache.getOrCreateWorkflowExecutionWithTimeout(ctx, domainID, workflowExecution)
+	if err != nil {
+		return nil, err
 	}
 	defer func() { release(retError) }()
 
 Update_History_Loop:
```

Comment:
Could this also be replaced with updateWorkflowExecutionWithAction?

Reply:
handling of decision task completed is really complex.
for now, i prefer stop here